README.md

```diff
@@ -1,193 +1,198 @@
+Wait for the installation to finish. It can take over ten minutes for everything to download and then start up.
 
-1. Verify the status of your cluster by running this command.
+Verify the status of your cluster by running:
 
-    ```sh
-    $ kubectl get pods -A
-    ```
-    If all the pods are listed as either `running` or `completed`, your cluster is ready for use.
+```sh
+$ kubectl get pods -A
+NAMESPACE     NAME                                      READY   STATUS      RESTARTS      AGE
+kube-system   coredns-96cc4f57d-jg2mh                   1/1     Running     0             10m
+kube-system   local-path-provisioner-84bb864455-fhfn6   1/1     Running     0             10m
+kube-system   metrics-server-ff9dbcb6c-wv7mt            1/1     Running     0             10m
+kube-system   helm-install-traefik-crd--1-sgwkd         0/1     Completed   0             10m
+kube-system   helm-install-traefik--1-z78sx             0/1     Completed   2             10m
+kube-system   svclb-traefik-qlkfb                       2/2     Running     0             9m16s
+kube-system   traefik-56c4b88c4b-zl7dq                  1/1     Running     0             9m16s
+loki          loki-0                                    0/1     Running     0             72s
+tempo         tempo-8649b7fb48-tmxz9                    1/1     Running     0             67s
+default       node-exporter-w77mf                       1/1     Running     0             59s
+tns           db-6f596588c5-k5zn4                       1/1     Running     2 (63s ago)   69s
+tns           loadgen-5dd787db94-2lnvc                  1/1     Running     2 (62s ago)   69s
+tns           app-85c6c6dc56-btcpk                      1/1     Running     2 (64s ago)   69s
+default       nginx-66fd8f67db-l9hnm                    1/1     Running     0             59s
+default       kube-state-metrics-575cf6678f-qkvph       1/1     Running     0             59s
+default       alertmanager-0                            2/2     Running     0             59s
+default       grafana-67554cdcf9-fv7bw                  1/1     Running     0             59s
+default       prometheus-0                              2/2     Running     0             59s
+default       grafana-agent-zl5hn                       1/1     Running     0             59s
 
-1. Access TNS using the URL [http://localhost:8080/](http://localhost:8080).
+```
 
-Note: If you need to re-do this process to get everything running, you can run `k3d cluster delete tns` to delete the cluster, then run `./create-k3d-cluster` and re-start the process. 
+If all the pods are listed as either `Running` or `Completed`, your cluster is ready for use.
 
-## Explore metrics to logs to traces
+You can access the demo at [http://localhost:8080/](http://localhost:8080), which will have links to Grafana and the other components.
 
-The following instructions will help you go from metrics to logs to traces.
```

Comment:
Why were these changed?

Reply:
The instructions themselves? As someone new to the ecosystem working through the demo for the first time, I found some of the instructions lacked the clarity necessary to easily navigate the demo. Some of them were also outdated (e.g.: 'select "Prometheus" as the datasource' should instead be "prometheus-exemplars" in order to see exemplar information). Are there specific changes you'd like made?